server/models/types.ts

```diff
@@ -88,26 +88,50 @@ export type GetDetectorsQueryParams = {
   sortField: string;
 };
 
+export type GetAdMonitorsQueryParams = {
+  from: number;
+  size: number;
+  search: string;
+  indices?: string;
+  sortDirection: SORT_DIRECTION;
+  sortField: string;
+};
+
 export type DetectorResultsQueryParams = {
   from: number;
   size: number;
   sortDirection: SORT_DIRECTION;
   sortField: string;
-  range?: object;
+  dateRangeFilter?: DateRangeFilter;
 };
 
 export type AnomalyResult = {
   startTime: number;
   endTime: number;
-  grade: number;
+  plotTime: number;
+  anomalyGrade: number;
   confidence: number;
 };
```

Comment:
do these changes break code anywhere else that uses these types?

Reply:
No, this only used by `getAnomalyResults`